packages/eas-cli/src/commands/branch/delete.ts

```diff
@@ -106,8 +106,8 @@ export default class BranchDelete extends EasCommand {
 
     const projectDir = await findProjectRootAsync();
     const exp = getExpoConfig(projectDir);
-    const fullName = await getProjectFullNameAsync(exp, { nonInteractive });
     const projectId = await getProjectIdAsync(exp, { nonInteractive });
+    const displayName = await getDisplayNameForProjectIdAsync(projectId);
```

Comment:
nit: maybe it would be worth unifying naming, and use projectDisplayName everywhere?